ledger/trie/branch.go

```diff
@@ -15,17 +15,19 @@
 package trie
 
 import (
+	"github.com/onflow/flow-go/ledger"
 	"github.com/onflow/flow-go/ledger/common/hash"
 )
 
 // Branch is a node is an intermediary node which has children.
 // It does not need to contain a path, because its children are ordered
 // based on their own path differences.
 type Branch struct {
-	hash  hash.Hash
-	dirty bool
-	left  Node
-	right Node
+	height uint16
```

Comment:
It is necessary to store the height of the branch in the branch to compute its hash, in a Flow Go trie implementation.